.ci-builds/.azure-exhaustive-tests.yml

```diff
@@ -71,4 +71,34 @@ jobs:
         CC: clang
         CXX: clang++
 #-------------------------------------------------------------------------
-
+# Based on https://unicode-org.github.io/icu/processes/release/tasks/integration.html#test-icu4j-with-only-little-endian-icu4c-data
+- job: ICU4J_with_only_little_endian_ICU4C_data
+  displayName: 'CJ: ICU4J Linux Clang Tests little-endian ICU4C data (Ubuntu 18.04)'
+  timeoutInMinutes: 50
+  pool:
+    vmImage: 'ubuntu-18.04'
+    demands: ant
+  steps:
+    - checkout: self
+      lfs: true
+      fetchDepth: 10
```

Comment:
1 ?

Reply:
Again, it is "fetchDepth: 10" in other builds in this file. I want to keep it as the same as the rest unless there is a good reason to change from it.